_episodes/02-scripts.md

```diff
@@ -105,13 +103,37 @@ This is another plus of writing scripts: you can comment
 your code to make it easier to understand when you come
 back to it after a while.
 
-We can combine multiple plots into one figure using the `subplot` command
-which plots our graphs in a grid pattern.
-The first two parameters describe the grid we want to use, while the third
-parameter indicates the placement on the grid.
+The ultimate form of writing comments is to write help text
```

Comment:
I'm not sure about this wording: "ultimate form". I also [hesitate](http://python-102.readthedocs.io/en/latest/documenting.html#docstrings-and-comments) to call documentation "comments"

Reply:
How about this?
> Help text is a special type of comment which acts as documentation for your script (or function)

Although help text has a specific purpose which is different to comments in general, it is most definitely a subset of comments (at least in MATLAB) - the lines start with a comment symbol :) MATLAB's [documentation](https://uk.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html) has this to say:

>Create help text by inserting comments at the beginning of your program

Reply:
Thanks for the comments - I've rewritten that section now and I think it's better for it - what do you think @shwina?